### PR TITLE
Updates to k8s quickstart for v3

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Removes the `calico-policy-controller` from the output...because it's been removed from the manifest. 
- Removes the apt-get update etc based on feedback. At this time, they appear to be unnecessary and redundant with main kubeadm install procedure.
- Removes join command, as not needed in single node install per @shatrugna .
- Fixes link syntax, which was incorrect and brittle.
- Removes "two-tier" which adds confusion with other types of tiers.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: master/getting-started/kubernetes/index.md [hunk 1]
```diff
@@ -42,6 +42,12 @@ the host. Instead, continue directly to the
    sudo apt-get update && sudo apt-get upgrade
    ```
    
+1. Restart Docker.
+
+   ```
+   sudo systemctl restart docker
```
Comment (ozdanborne) [inline master/getting-started/kubernetes/index.md]: Do we know exactly why is this needed? And are we sure it is needed every time?

Comment (emanic) [reply master/getting-started/kubernetes/index.md]: It clears up failures that can occur after dockerd gets updated. It might not be needed every time, but there's no harm in executing it and executing it means the difference between success and failure in the procedure....so it creates a more robust and reliable experience for the readers. Thanks!

Comment (ozdanborne) [reply master/getting-started/kubernetes/index.md]: IMO putting it in the instructions makes it seem like Calico installation requires restarting the docker daemon, which it does not. I'm strongly against mentioning this at all.

Comment (emanic) [reply master/getting-started/kubernetes/index.md]: Not sure I understand your concern....can we find some time to discuss more in person? Thanks @ozdanborne

Comment (matthewdupre) [reply master/getting-started/kubernetes/index.md]: My personal preference would be to ditch both this and the apt-get update / upgrade lines (which are Ubuntu specific anyway even though this doc supports other distros).

IMO the only excuse to suggest running this kind of thing is if you've added an apt source.

Comment (emanic) [reply master/getting-started/kubernetes/index.md]: The reason I had the apt-get update/upgrade bit is because it appeared in the source. It does add a lot of time to the procedure and other people have commented about this. (Discussions of other distros is something I have on my "to do".) Looking into this more, I see that that main kubeadm install page also has them running this command just prior so it's pretty redundant. I'm OK dropping these two.
